### PR TITLE
update restore-from-backup script

### PR DESCRIPTION
**Pull request by Dhruv-Pattel**
solved the issue #112

### CODE REVIEW

File: scripts/restore-from-backup.sh [hunk 1]
```diff
@@ -6,3 +6,12 @@ source .env
 MYSQL_CONTAINER=$(docker ps --format '{{ .Names }}' | grep mysql)
 docker cp $1 $MYSQL_CONTAINER:metakgp_wiki_db.sql
 docker exec $MYSQL_CONTAINER sh -c 'mysql -u metakgp_user -p'$MYSQL_PASSWORD' metakgp_wiki_db < metakgp_wiki_db.sql'
+
+result=$(find /backup -type f -name "images")
```
Comment (harshkhandeparkar) [inline scripts/restore-from-backup.sh]: Can you rename this to something more descriptive like images_exist? Also shouldn't the command be `find ./backup`?

**Comment by Dhruv-Pattel** [comment]
I have made the required changes. Check that out.

**Comment by harshkhandeparkar** [comment]
Have you tested the script?

**Comment by Dhruv-Pattel** [comment]
I tested the script without running the containers (cause I was getting errors while running containers as mentioned in the documentation).

error while running container 👇
container

running script 👇
bash

**Comment by harshkhandeparkar** [comment]
> I tested the script without running the containers (cause I was getting errors while running containers as mentioned in the documentation).
> 
> error while running container 👇 container
> 

Can you send a longer screenshot? We could also help you debug faster if you join our [Slack](https://slack.metakgp.org).

**Review by shikharish — COMMENTED** [review]
The script should take the path to a tar.gz file(the one downloaded from cloud backup) as an argument. So we would have to extract the tar file first, then copy the sql file to the container, check if there is an images folder and copy that folder too.

**Pull request closed without merging**